### ISSUE TITLE
import bz2 fails with Symbol not found.

### ISSUE THREAD

**Issue by joernhees** [joernhees/sublime-compressor#1]
```
reloading plugin Compressor.Compressor
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 73, in reload_plugin
    m = importlib.import_module(modulename)
  File "./importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 584, in _check_name_wrapper
  File "<frozen importlib._bootstrap>", line 1022, in load_module
  File "<frozen importlib._bootstrap>", line 1003, in load_module
  File "<frozen importlib._bootstrap>", line 560, in module_for_loader_wrapper
  File "<frozen importlib._bootstrap>", line 868, in _load_module
  File "<frozen importlib._bootstrap>", line 313, in _call_with_frames_removed
  File "/Users/joern/Library/Application Support/Sublime Text 3/Packages/Compressor/Compressor.py", line 4, in <module>
    import bz2
  File "./bz2.py", line 21, in <module>
ImportError: No module named '_bz2'
```

**Comment by dvhh** [comment]
according to [various][1] [sources][2], issue came from how the python interpreter shipped with sublime text was compiled, this no longer appear to be the issue with sublime text 3.

After some digging, the modules was added in in build 3114 of sublime text 3.

[1]:https://stackoverflow.com/questions/38630720/python-import-error-no-module-named-bz2
[2]:https://stackoverflow.com/questions/8115280/importerror-no-module-named-bz2-for-python-2-7-2

**Comment by joernhees** [comment]
closed in #2
